spec/features/visitors/home_page_spec.rb

```diff
@@ -1,14 +1,12 @@
-# Feature: Home page
-#   As a visitor
-#   I want to visit a home page
-#   So I can learn more about the website
-feature 'Home page' do
+require 'rails_helper'
 
-  # Scenario: Visit the home page
-  #   Given I am a visitor
-  #   When I visit the home page
-  #   Then I see "Welcome"
-  scenario 'visit the home page' do
+feature 'Home page', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.